int8 id 		#id number of servo [0, 7]
float32 angle 	#angle in degrees, must be between 0 and 180